knowledge-base/v4-api.md

```diff
@@ -79,8 +80,7 @@ This document contains a collection of OData v4 features we want to implement an
   ```
 
 ### Decision:
-- Second proposal accepted under the assumption that expanding navigation properties automatically selects all properties of the navigation property.
-- If the assumtion is incorrect, use the first proposal.
+- First proposal accepted. Our assumption that expanding navigation properties automatically selects all properties of the navigation property has proven correct.
```

Comment:
If our assumption is correct, shouldn't the second one then be the accepted one, not the first?

Reply:
👍 also confused. Question which just came to my mind. What happens for nested navigation properties so if the expanded one has again a navigation property? I guess this is not expanded, right?

Reply:
It can also be expanded. And of course you are right, I confused the order.